BinaryObjectScanner.Test/Protection/InterLokTests.cs

```diff
@@ -0,0 +1,22 @@
+using System.IO;
+using BinaryObjectScanner.Protection;
+using Xunit;
+
+namespace BinaryObjectScanner.Test.Protection
+{
+    public class InterLokTests
+    {
+        [Fact]
+        public void CheckPortableExecutableTest()
+        {
+            string file = "filename";
+            SabreTools.Models.PortableExecutable.Executable model = new();
```

Comment:
Quickly double check if I used `var` syntax or `new()` syntax in other test classes. Whichever it is, I'd like to keep consistent to. If this is copy-pasted and then only tweaked to update the class name, then just ignore this.